ledger/participant-integration-api/src/main/scala/platform/store/packagemeta/PackageMetadataView.scala

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.packagemeta
+
+import com.daml.lf.archive.Decode
+import com.daml.lf.data.Ref
+import com.daml.daml_lf_dev.DamlLf
+import com.daml.platform.store.packagemeta.PackageMetadataView._
+import scalaz._
+
+trait PackageMetadataView {
+  def update(metadataDefinitions: MetadataDefinitions): Unit
+
+  def read(): PackageMetadata
+}
+
+object PackageMetadataView {
+  def create: PackageMetadataView = new PackageMetaDataViewImpl
+
+  trait PackageMetadata {
```

Comment:
Is this trait necessary at all? Can't we just deal in MetadataDefinitions business? (eg dropping this, and just returning MetadataDefinitions from PackageMetadataView.read())